test/endpoint.test.ts

```diff
@@ -159,10 +159,64 @@ describe("endpoint()", () => {
       headers: {
         authorization: `token 0000000000000000000000000000000000000001`,
         accept: "application/vnd.github.v3+json",
-        "content-length": 0,
         "user-agent": userAgent,
       },
-      body: "",
```

Comment:
I confirmed that the `"content-length": 0` is no longer needed for the `PUT /user/starred/{owner}/{repo}` endpoint.